src/main/common/filter.c

```diff
@@ -302,3 +302,33 @@ float firFilterDenoiseUpdate(firFilterDenoise_t *filter, float input)
         return filter->movingSum / ++filter->filledCount + 1;
     }
 }
+
+// Fast two-state Kalman
```

Comment:
Is this really two-state kalman? We are only estimating one state that is gyro rate